### ISSUE TITLE
addInlineJs and addInlineCss arent working

### ISSUE THREAD

**Issue by Dimous** [phalcon/cphalcon#10533]
Windows 8  64-bit
XAMPP Version: 5.6.8
phalcon_x86_vc11_php5.6.0_2.0.3

```php
class BaseController extends \Phalcon\Mvc\Controller {
 public function initialize() {
        $this->assets->addInlineJs("alert('test')")->addJs("static/js/script.js")->addCss("static/css/normalize.css")->addCss("static/css/style.css");
 }
...
} 
```

addInlineJs cause "Fatal error: Call to a member function addJs() on null"
without addInlineJs its working

in serial calls:
```php
$this->assets->addInlineJs("alert('test')");
$this->assets->addJs("static/js/script.js")->addCss("static/css/normalize.css")->addCss("static/css/style.css");
```
alert code is never added to the view

**Comment by virgofx** [comment]
Looks like both **addInlineCss** and **addInlineCss** do not include the required chaining `return this` return value.

**Comment by andresgutierrez** [comment]
Fixed in the 2.0.x branch
